core/handlers.py

```diff
@@ -217,6 +216,11 @@ def sv_post_load(scene):
 
     for ng in sv_trees:
         if ng.bl_idname == 'SverchCustomTreeType' and ng.nodes:
+
+            for node in ng.nodes:
+                if hasattr(node, "is_animatable"):
+                    node.process()
```

Comment:
From my point of veiw only the update system can call the process method because the method of nodes should be called in proper order to get input data, otherwise the method can raise NoData error easily.

Reply:
i know.

Reply:
this PR is not intended to be pushed. at all. ever.

Reply:
I wonder what is purpose of such PRs?